app-multimedia/qpwgraph/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=qpwgraph
+PKGDES="A PipeWire graph manager"
```

Comment:
A graph-based PipeWire topology manager

Reply:
I think the project calls itself "graph manager" in README, see https://github.com/rncbc/qpwgraph